feature/policy_forwarding/encapsulation/otg_tests/staticgueencap_and_bgp_path_selection/README.md

```diff
@@ -0,0 +1,382 @@
+	    * ATE1:Port1 to ATE2:Port1 flows are expected to be 100% successful and routing over the IBGP peering between DUT:Port2 and ATE2:Port1
+	    * ATE2 to ATE1:Port1 Unencaped flows must be 100% successful. The flows are expected to route through the IBGP peering between ATE2:Port1 --> DUT:Port2 --> ATE
+	    * Run post test Healthchecks and compare. Ensure there are no drops, coredumps etc.<br><br><br>
+
+
+**PF-1.6.2: Migrate routing of BE1 flows from ATE1:Port1 to ATE2:Port1 to ATE2:Port3 instead**
+   * Situation:
+       * The IBGP session between ATE2:Port2 and DUT:Port3 is up and advertising only IPv4Prefix6/24, IPv6Prefix6/64. Hence now traffic to these destination prefixes must be GUE encaped and sent over the EBGP peering between DUT:Port4 and ATE2:Port3
+       * Unencaped traffic received from ATE2 to ATE1:Port1 is routed via the IBGP session between DUT:Port2 and ATE2:Port1
+   * Test Steps
+       * Please run all the health checks suggested above prior to starting the test as a baseline.
+       * Configure DUT as suggested above.
+           * Make sure the IBGP peering between DUT:Port3 and ATE2:Port2 is up and advertising only IPv4Prefix6/24, IPv6Prefix6/64 with a VPNH as explained in the `Advertisements` section above.
+       * Start above mentioned flows
+           * Dont start the above mentioned encaped flows from ATE2:Port3
+       * The Traffic that gets encaped, Ensure the TTL value of the inner header is decremented and copied to the outer header. Similarly, TOS byte in the inner header is copied to the Outer header.<br>
+         
+   * Expectations:
+       * Routes to Prefixes IPv4Prefix6/24, IPv6Prefix6/64 learnt from ATE2:Port2 will have a Local-Preference of 200 and hence placed in the FIB. Routes to other prefixes from ATE2 will be learnt from the IBGP peering with ATE2:Port1, will continue to have the default local-Preference of 100 and will be placed in the DUT's FIB  
+       * Flows to destination prefixes IPv4Prefix6/24, IPv6Prefix6/64 will be GUE encaped to tunnel destination IPv4Prefix12/28. the flows must be successful.
+           * Ensure that the TTL on the outer heder is decremented and lower than that of the inner header.
+           * TOS byte in the outer header is the same as the TOS value in the inner header.
+           * Ensure the DUT is streaming accurate data on the number of packets/bytes being Encaped.
+	   * ATE2 to ATE1:Port1 Unencaped flows must be 100% successful. The flows are expected to route through the IBGP peering between ATE2:Port1 --> DUT:Port2 --> ATE
+	   * Run post test Healthchecks and compare. Ensure there are no drops, coredumps etc.<br><br><br>
+
+
+**PF-1.6.3: Migrate routing of AF1-AF4 flows from ATE1:Port1 to ATE2:Port1 to ATE2:Port3 instead**
+   * Follow the same steps as above in PF-1.6.2 and gradually move one Traffic class at a time in the following order
+       * PF-1.6.3: Migrate routing of AF1 flows from ATE1:Port1 to ATE2:Port1 to ATE2:Port3 instead
```

Comment:
Can we clarify the implementation here? It seems like we are doing this by advertising a particular prefix one at a time to on the session @ ATE2:port3, right?

We should clarify what steps in 1.6.2 we are covering here -- since migrating per AFI isn't defined it's not really clear what to do here (I think you're adding to the iBGP session advertisements, but it's not clearly specified).